spec/passaporte_web/http_spec.rb

```diff
@@ -13,7 +13,7 @@
     end
   end
 
-  let(:mock_response) { mock('restclient http response') }
+  let(:mock_response) { 'restclient http response' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.